Mathlib/Tactic/ToAdditive/Frontend.lean

```diff
@@ -1183,9 +1183,11 @@ partial def applyAttributes (stx : Syntax) (rawAttrs : Array Syntax) (thisAttr s
   if linter.existingAttributeWarning.get (← getOptions) then
     let appliedAttrs ← getAllSimpAttrs src
     if appliedAttrs.size > 0 then
+      let appliedAttrs := ", ".intercalate (appliedAttrs.toList.map toString)
+      -- Note: we're not bothering to print the correct attribute arguments.
```

Comment:
Could you clarify this comment? I'm not quite sure I understand

Reply:
If someone writes `@[simp↓, to_additive]`, then the suggestion will be `@[to_additive (attr := simp)]`, forgetting the `↓`. Since this is just a suggestion, I don't think this is a big deal.